lib/index.js

```diff
@@ -35,7 +37,7 @@ function filterKey(key) {
 // Public interfaces
 //------------------------------------------------------------------------------
 
-module.exports = Object.freeze({
+export default Object.freeze({
```

Comment:
Would named exports make more sense than `export default`, since `KEYS`, `getKeys()` and `unionWith()` look more like separate features than properties of an object?

For example, typescript-eslint imports this library with [`import * as eslintVisitorKeys from 'eslint-visitor-keys'`](https://github.com/typescript-eslint/typescript-eslint/blob/8cfe93372e1d826e54febc3aeb7047c792b90963/packages/visitor-keys/src/visitor-keys.ts#L2).

Reply:
Yeah, this is an issue to decide on the other repos needing changing as well. I think separate is good to allow for tree-shaking, but y'all's call. Let me know.

Reply:
Let's do named exports, without `export default`.

Reply:
> Let's do named exports, without `export default`.

To be clear, this is only about `lib/index.js`. 

`export default` is okay in `lib/visitor-keys.js`

Reply:
Do you want the exports at the end of the file or inline?

Reply:
Maybe inline, like in espree?

Reply:
Ok, done.